x-pack/filebeat/module/virustotal/livehunt/_meta/fields.yml

```diff
@@ -0,0 +1,924 @@
+            Type of imported symbol
+          type: keyword
+          default_field: false
+    - name: shared_libraries
+      description: >
+        List of shared libraries used by this ELF object
+      type: keyword
+    - name: telfhash
+      description: >
+        telfhash hash for ELF files.
+      type: keyword
+    - name: flattened
+      default_field: false
+      description: >
+        Flattened ELF events from VirusTotal Intelligence Live Hunt results.
+      release: beta
+      type: group
+      fields:
+        - name: segment_list
+          description: >
+            ELF object segment list.
+          type: flattened
+- name: file.pe
+  type: group
+  fields:
+    - name: flattened
+      release: beta
+      type: group
+      fields:
+        - name: debug
+          default_field: false
+          description: >
+            Debug information, if present
+          type: flattened
+        - name: import_list
+          default_field: false
+          description: >
+            List of all imported functions
+          type: flattened
+        - name: sections
+          default_field: false
+          description: >
+            Data about sections of compiled binary PE
+          type: flattened
+        - name: resource_details
+          default_field: false
+          type: flattened
+          description: >
+            If the PE contains resources, some info about them
+        - name: resource_languages
+          default_field: false
+          type: flattened
+          description: >
+            Digest of languages found in resources. Key is language (as string) and
+            value is how many resources there are having that language (as integer)
+        - name: resource_types
+          default_field: false
+          type: flattened
+          description: >
+            Digest of resource types. Key is resource type (as string) and value
+            is how many resources there are of that specific type (as integer)
+        - name: packers
+          default_field: false
+          type: flattened
+          description: >
+            Identifies packers used on Windows PE files by several tools and AVs.
+            Keys are tool names and values are identified packers, both strings.
+            see `file.pe.packers` for merged list of packers from all tools.
+      overwrite: true
+    - name: exports
+      type: keyword
+      description: >
+        List of symbols exported by PE
+
+    - name: creation_date
+      default_field: false
+      description: >
+        extracted when possible from the file's metadata. Indicates when it was
+        built or compiled. It can also be faked by malware creators.
+      type: date
+    - name: authentihash
+      description: >
+        Authentihash of the PE file.
+      type: keyword
+    - name: compile_timestamp
+      description: >
+        Compile timestamp of the PE file.
+      type: date
+
+    - name: packers
```

Comment:
why have this and the `flattened` field?

Reply:
This can probably actually be removed. I restructured `virustotal.packers` because the data returned is consistent for both ELFs and PEs to include the analysis tool name and the resulting value. This isn't what the docs said though, so this was an attempt to provide a consistent interface with ELF data. I'll axe it.